include/mbedtls/pk.h

```diff
@@ -84,6 +84,18 @@
 extern "C" {
 #endif
 
+#if defined(MBEDTLS_USE_PSA_CRYPTO)
+int mbedtls_pk_psa_err_translate( psa_status_t status );
```

Comment:
These functions are meant for internal use and I don't think we should guarantee their API stability. So they should be declared in `library/*.h`.

Reply:
@gilles-peskine-arm would a new header named `library/psa_crypto_wrap.h` be ok for these, or multiple with one per module ?

Reply:
If they're `mbedtls_pk_xxx` functions, then I think the existing `pk_wrap.h` makes sense.